src/hooks/useCombobox/index.js

```diff
@@ -65,6 +68,7 @@ function useCombobox(userProps = {}) {
   const isInitialMount = useRef(true)
   const elementIdsRef = useRef(getElementIds(props))
   const previousResultCountRef = useRef()
+  const latest = useLatestRef({state, props})
```

Comment:
This just exists so we can use `latest` in our callbacks without worrying about these values breaking memoization.